src/main/scala/akka/persistence/jdbc/journal/JdbcAsyncWriteJournal.scala

```diff
@@ -33,6 +33,9 @@ import scala.collection.immutable._
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.{Failure, Success, Try}
 
+object JdbcAsyncWriteJournal {
+  val Identifier = "jdbc-journal"
```

Comment:
Cool, that I didn't know. However I'll remove the companion object as I'll switch to use configuration instead. But thanks for the information.